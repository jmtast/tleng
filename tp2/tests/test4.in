// Error de semántica: más de 16 voces
#tempo negra 120
#compas 2/4

const oct1=5;

voz (0)
{
  compas
  {
    nota(re, oct1, blanca);
  }
}
voz (0)
{
  compas
  {
    nota(re, oct1, blanca);
  }
}
voz (0)
{
  compas
  {
    nota(re, oct1, blanca);
  }
}
voz (0)
{
  compas
  {
    nota(re, oct1, blanca);
  }
}
voz (0)
{
  compas
  {
    nota(re, oct1, blanca);
  }
}
voz (0)
{
  compas
  {
    nota(re, oct1, blanca);
  }
}
voz (0)
{
  compas
  {
    nota(re, oct1, blanca);
  }
}
voz (0)
{
  compas
  {
    nota(re, oct1, blanca);
  }
}
voz (0)
{
  compas
  {
    nota(re, oct1, blanca);
  }
}
voz (0)
{
  compas
  {
    nota(re, oct1, blanca);
  }
}
voz (0)
{
  compas
  {
    nota(re, oct1, blanca);
  }
}
voz (0)
{
  compas
  {
    nota(re, oct1, blanca);
  }
}
voz (0)
{
  compas
  {
    nota(re, oct1, blanca);
  }
}
voz (0)
{
  compas
  {
    nota(re, oct1, blanca);
  }
}
voz (0)
{
  compas
  {
    nota(re, oct1, blanca);
  }
}
voz (0)
{
  compas
  {
    nota(re, oct1, blanca);
  }
}
voz (0)
{
  compas
  {
    nota(re, oct1, blanca);
  }
}
