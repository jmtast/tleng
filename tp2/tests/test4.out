Error:
No se permiten más de 16 voces. Línea 119
